src/js/modules/infragistics.datasource.js

```diff
@@ -5269,12 +5285,26 @@
 						}
 					}
 					if (methodsStr !== "") {
-						params.summariesParams[ s.summaryExprUrlKey + "(" + cs[ i ].columnKey + ")" ] =
-							methodsStr.slice(0, -1);
+						fieldName = cs[ i ].columnKey;
+						field = this._getSchemaField(fieldName);
+						if (field && field.type) {
+							key = s.summaryExprUrlKey + "(" + fieldName + ":" + field.type + ")";
+						} else {
+							key = s.summaryExprUrlKey + "(" + fieldName + ")";
+						}
+
+						params.summariesParams[ key ] = methodsStr.slice(0, -1);
 					}
 				}
 			}
 		},
+		_getSchemaField: function (fieldName) {
+			if (this.settings.schema && this.settings.schema.fields.find) {
+				return this.settings.schema.fields.find(function (f) { return f.name === fieldName; });
```

Comment:
@bkulov : Array.prototype.find is not available in Internet Explorer. This code will throw an error there. I'm not sure we provide a polyfill for it. Can you check it please.

Reply:
Sure. Will modify it.

Reply:
@bkulov Agree. It won't throw and error.

Reply:
Done :)